test/end-to-end/test_make_screenshots.sh

```diff
@@ -22,6 +22,9 @@ phantomjs --ignore-ssl-errors=true --web-security=false ${DIR}/make_screenshot.j
                               "${DIR}/screenshots/study_view_lgg_ucsf_2014.png" \
                               5000
 
+# upload the second image
+curl -F "clbin=@${DIR}/screenshots/study_view_lgg_ucsf_2014.png" https://clbin.com							  
```

Comment:
The second image does not need to be uploaded, because in the statements below you can see that it is actually only uploaded in case the images differ (using `git diff`).